package.json

```diff
@@ -20,9 +39,9 @@
     "storybook": "run-s build:icons:index storybook:dev",
     "storybook:dev": "start-storybook --port 9001 --config-dir .storybook --static-dir ./public",
     "build-storybook": "build-storybook --config-dir .storybook --output-dir .out --static-dir ./public",
-    "clean": "rimraf dist lib src/icons/**/svgs",
-    "build": "run-s clean && run-s build:icons:all && run-s tsc && run-p build:rollup build:babel && run-p bundlewatch",
-    "build:icons:all": "run-s clean && run-s build:icons build:icons:index",
+    "clean": "rimraf dist src/icons/**/svgs",
+    "build": "run-s clean build:icons build:icons:index build:rollup bundlewatch",
+    "build:icons:all": "run-s build:icons build:icons:index",
```

Comment:
I think we should keep clean in `build:icons:all`, unless you saw some issue. There's a manual step involved with making icons which is why it exists at all, instead of just relying on `build`.

Reply:
Oh that was a mistake, it's supposed to be in there. I was trying to remove the redundant call to clean when running `build`.

Reply:
Fixed this in bb2c6d5